subsys/bluetooth/controller/ticker/ticker.h

```diff
@@ -64,16 +64,24 @@
 #if defined(CONFIG_BT_TICKER_EXT)
 #if defined(CONFIG_BT_TICKER_SLOT_AGNOSTIC)
 #define TICKER_USER_OP_T_SIZE   44
-#else
+#else /* !CONFIG_BT_TICKER_SLOT_AGNOSTIC */
+#if defined(CONFIG_BT_TICKER_REMAINDER)
+#define TICKER_USER_OP_T_SIZE   52
+#else /* !CONFIG_BT_TICKER_REMAINDER */
 #define TICKER_USER_OP_T_SIZE   48
-#endif /* CONFIG_BT_TICKER_SLOT_AGNOSTIC */
-#else
+#endif /* !CONFIG_BT_TICKER_REMAINDER */
+#endif /* !CONFIG_BT_TICKER_SLOT_AGNOSTIC */
+#else /* !CONFIG_BT_TICKER_EXT */
 #if defined(CONFIG_BT_TICKER_SLOT_AGNOSTIC)
 #define TICKER_USER_OP_T_SIZE   40
-#else
+#else /* !CONFIG_BT_TICKER_SLOT_AGNOSTIC */
+#if defined(CONFIG_BT_TICKER_REMAINDER)
+#define TICKER_USER_OP_T_SIZE   48
+#else /* !CONFIG_BT_TICKER_REMAINDER */
 #define TICKER_USER_OP_T_SIZE   44
-#endif /* CONFIG_BT_TICKER_SLOT_AGNOSTIC */
-#endif /* CONFIG_BT_TICKER_EXT */
+#endif /* !CONFIG_BT_TICKER_REMAINDER */
+#endif /* !CONFIG_BT_TICKER_SLOT_AGNOSTIC */
+#endif /* !CONFIG_BT_TICKER_EXT */
```

Comment:
These are manually calculated struct sizes, and ticker.c file has build asserts to verify them. For historical reason, user of the ticker module (in case prebuilt library) was to supply the memory at runtime and use these size to allocate memory in the upper layers.